index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"/>
+
+	<title>My Portifolio Page</title>
+	<link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+<body>
+	<div class="main">
+		<section class="toolbar">
```

Comment:
Let's make this `section` tag a `header`. It would make much more sense semantically and also that this section is actually a header. If you are not familiar with semantic HTML tags, have a look at [this](https://www.w3schools.com/html/html5_semantic_elements.asp).